best-practices/README.md

```diff
@@ -110,6 +110,10 @@ Testing
 
 * Avoid `any_instance` in rspec-mocks and mocha. Prefer [dependency injection].
 * Avoid `its`, `let`, `let!`, `specify`, and `before` in RSpec.
+  * Instead of using these, extract a method that you can pass the thing that
```

Comment:
This point only applies to `let` and `let!`. It could also apply to using `before` to set an instance variable. I've also noticed that `let` is used poorly specifically when building objects or factories. So, what about:

> Using `let` (or `let!`) particularly to build objects prevents those objects from being customized in tests that need to so. Instead, prefer extracting a helper method.